requirements/edx/custom.txt

```diff
@@ -22,7 +22,7 @@ git+https://github.com/edx-solutions/xblock-group-project.git@0.1.2#egg=xblock-g
 -e git+https://github.com/mckinseyacademy/xblock-diagnosticfeedback.git@v0.2.4#egg=xblock-diagnostic-feedback==0.2.4
 -e git+https://github.com/open-craft/xblock-group-project-v2.git@0.4.13#egg=xblock-group-project-v2==0.4.13
 -e git+https://github.com/open-craft/xblock-virtualreality.git@v0.1.1#egg=xblock-virtualreality==0.1.1
-git+https://github.com/edx-solutions/api-integration.git@v2.5.4#egg=api-integration==2.5.4
+git+https://github.com/edx-solutions/api-integration.git@v2.5.5#egg=api-integration==2.5.5
```

Comment:
your release tag is `2.5.5` in `api-integration` while here you are using `v2.5.5`, it should match the tag in `api-integration`

Reply:
I have update the release tag in the api-integration